app/pages/_document.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Document, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+} from 'next/document';
+
+function Analytics() {
+  return (
+    <>
+      <script async src={`https://www.googletagmanager.com/gtag/js?id=${process.env.GOOGLE_ANALYTICS}`} />
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+            window.dataLayer = window.dataLayer || [];
+            function gtag() {
+              dataLayer.push(arguments);
+            }
+            gtag('js', new Date());
+            gtag('config', ${process.env.GOOGLE_ANALYTICS});
```

Comment:
I've specify GOOGLE_ANALYTICS=helloworld and got:
![image](https://user-images.githubusercontent.com/14346393/115375535-903e3000-a1d6-11eb-88ef-958088491959.png)

It seems `""` are missing.